data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,689 @@ mission "Remnant: Learn Sign Follow Up"
+			`	"Aside from the strange hull structure, these ships also seem to have a considerably amount of equipment integrated directly into the ships themselves. In-built scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of in-built device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`
+			`	"Do you think I could get a copy of your research?" you ask.`
+				goto suspicion
+			
+			label technology
+			choice
+				`	"I was wondering what your research on Remnant technology has brought up."`
+				`	"Could I get a copy of your research on Remnant technology?"`
+					goto suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size, more so than even atomic engines of a similar size, while not producing very much heat at all. Their major downside though seems to be the immense amount of energy they draw. The energy drain is not surpising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is somethign spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`
+			
+			branch skip
+				not "Deep: Remnant: Generators: done"
+			`	He pulls out a schematic showing the generator. "The generator on the other hand wasn't all too interesting. Little more than an improved generator of the type you might find down in the Rim. Somewhat interesting that it doesn't produce much energy when compared to something like a fusion reactor given the energy requirement of their engines, but I suppose this people are concerned more with keeping their ships running cool then having excess energy. The one interesting characteristic we discovered though is that the generator seemed to handle the effects of ionization, like something the Hai ion cannons apply, much better than normal. It seemed to be an intentional result of how the generator was designed."`
```

Comment:
Is there a check earlier that the player has met the Hai? I haven't seen one, but I'm only looking at the new stuff so it may have been earlier.

Reply:
The player must have met the Hai to even get to this point as a function of the offer requirements for previous missions.

Reply:
Sweet, just wanted to check.